docs/about/overview/encryption.mdx

```diff
@@ -72,3 +138,14 @@ I'm assuming that meshtastic is being used to hike in places where someone capab
 - Two other things to keep in mind are that AES-CTR does not itself provide authenticity (e.g. an attacker can flip bits in replaying data and scramble the resulting plaintext), and that the current scheme gives some hints about transmission in the size. So, if you worry about an adversary deliberately messing-up messages or knowing the length of a text message, it looks like those might be possible.
 
 I'm guessing that the network behaves somewhat like a store-and-forward network - or, at least, that the goal is to avoid establishing a two-way connection to transmit data. I'm afraid I haven't worked with mesh networks much, but remember studying them briefly in school about ten years ago.
+
+### Comments from @Jorropo
+
+- The IV initialization only use 31 random bits per reboot, it then increment messages sequentially. This is not high, it makes duplicate unlikely but probable given the complete scale of the network. Thankfully IV include lower 32 bits of the MAC address which should be unique per node, so each node is it's own birthday paradox, we aren't looking for network wide birthday paradox which would be easy to hack. See [`#4031`](https://github.com/meshtastic/firmware/issues/4031).
+  Could be fixed.
+- The lack of [integrity wasn't properly considered](https://github.com/meshtastic/firmware/issues/4030), the remote administration module implements priviliged RPC calls over AES-CTR without any MAC or AEAD.
+  Could be fixed.
+- The AES-CTR implementation looks like it protects Confidentiality assuming IVs are not duplicated.
+- The lack of Forward Secrecy is a bit worying for a chat messenger when we new users lack the paranoid and anoying key management practice that would be required for safe operation.
+- IMO the clients should have a big red box the first time you open a Direct-Message indicating theses are not private and not E2E. This was not clear to me without reading the code.
+- This project is completely fine for chating with strangers, using `AQ==` and understanding everything is public, I would not trust it with anything private without some other layer or significant reworks. For example I would consider SSH or Signal over Meshtastic safe. Yet Meshtastic does not choose to have worst crypto for bad reasons, they make the network significantly more usable over the unreliable slow LoRa backhaul it is using.
```

Comment:
- chat**t**ing
- I would replace `AQ==` with "the default key".
- worst -> worse